src/components/Home.css

```diff
@@ -0,0 +1,139 @@
+.gridList {
+  display: grid;
+  grid-template-columns: repeat(2, 50%);
+  width: 100%;
+  place-self: center;
+  padding: 0;
+}
```

Comment:
- [ ] Hey there, great job on setting up your page layout using grid! 🙌 However, I did notice a small issue that can impact user experience, especially on screens with a width of 576px. 📱

The problem is that some elements are overflowing outside of their containers, and this can make the website look a bit messy. 🙈 Specifically, I noticed this happening with the navbar and possibly other elements as well.

To ensure a seamless user experience on different screen sizes, it's crucial to make your design responsive. 🌐 Here are a few steps you can consider taking:

1. **Media Queries**: Use CSS media queries to adjust the layout and styling of your elements based on the screen width. You can define different CSS rules for screens of various sizes, ensuring that your elements fit neatly within their containers.

2. **Responsive Units**: Instead of fixed pixel values for widths and heights, consider using relative units like percentages or `vw` (viewport width) and `vh` (viewport height) to make your elements adapt to different screen sizes.

3. **Flexbox**: If you're not already using it, consider incorporating Flexbox alongside Grid for more control over the layout and alignment of your elements, especially in smaller screen sizes.

4. **Testing**: Don't forget to regularly test your website on various devices and screen sizes to ensure everything looks and works as expected.

Responsive design is essential for delivering a consistent and user-friendly experience across different devices, and it's a key aspect of modern web development. 🌟 Keep up the good work, and by making these adjustments, your website will shine on all screens! 💪🌐📱

see the below image of your current implementation
![Screenshot from 2023-09-16 04-28-09](https://github.com/Ness88/Air-Quality-Index-Tunisia/assets/112261237/18f5e32c-c5bf-4985-a0b4-07d4f4376648)